Tests/Data/Mechanics/MohrCoulombAbboSloanAniso/run_script.ipynb

```diff
@@ -0,0 +1,506 @@
+      "Generating input for 10 °\n",
+      "Running simulation for 10 °\n",
+      "Generating input for 13 °\n",
+      "Running simulation for 13 °\n",
+      "Generating input for 17 °\n",
+      "Running simulation for 17 °\n",
+      "Generating input for 20 °\n",
+      "Running simulation for 20 °\n",
+      "Generating input for 23 °\n",
+      "Running simulation for 23 °\n",
+      "Generating input for 27 °\n",
+      "Running simulation for 27 °\n",
+      "Generating input for 30 °\n",
+      "Running simulation for 30 °\n",
+      "Generating input for 33 °\n",
+      "Running simulation for 33 °\n",
+      "Generating input for 37 °\n",
+      "Running simulation for 37 °\n",
+      "Generating input for 40 °\n",
+      "Running simulation for 40 °\n",
+      "Generating input for 43 °\n",
+      "Running simulation for 43 °\n",
+      "Generating input for 47 °\n",
+      "Running simulation for 47 °\n",
+      "Generating input for 50 °\n",
+      "Running simulation for 50 °\n",
+      "Generating input for 53 °\n",
+      "Running simulation for 53 °\n",
+      "Generating input for 57 °\n",
+      "Running simulation for 57 °\n",
+      "Generating input for 60 °\n",
+      "Running simulation for 60 °\n",
+      "Generating input for 63 °\n",
+      "Running simulation for 63 °\n",
+      "Generating input for 67 °\n",
+      "Running simulation for 67 °\n",
+      "Generating input for 70 °\n",
+      "Running simulation for 70 °\n",
+      "Generating input for 73 °\n",
+      "Running simulation for 73 °\n",
+      "Generating input for 77 °\n",
+      "Running simulation for 77 °\n",
+      "Generating input for 80 °\n",
+      "Running simulation for 80 °\n",
+      "Generating input for 83 °\n",
+      "Running simulation for 83 °\n",
+      "Generating input for 87 °\n",
+      "Running simulation for 87 °\n",
+      "Generating input for 90 °\n",
+      "Running simulation for 90 °\n"
+     ]
+    }
+   ],
+   "source": [
+    "for i in beta:\n",
+    "    print(\"Generating input for %i °\"%(int(np.round(i*180/np.pi,0))))\n",
+    "    filename = \"triax_1e0_\"+str(int(np.round(i*180./np.pi,0)))+\".prj\"\n",
+    "    cmd = \"sed 's/<values>1 0/<values>\" + str(np.cos(i)) + \" \" + str(np.sin(i))+ \"/g' triax_original.prj > \" + filename\n",
+    "    os.system(cmd)\n",
+    "    cmd = \"sed 's/<values>0 1/<values>\" + str(-np.sin(i)) + \" \" + str(np.cos(i))+ \"/g' \" + filename + \" -i\"\n",
+    "    os.system(cmd)\n",
+    "    cmd = \"sed 's/<prefix>triax_0/<prefix>triax_\" +str(int(np.round(i*180./np.pi,0))) + \"/g' \" + filename + \" -i\"\n",
+    "    os.system(cmd)\n",
+    "    print(\"Running simulation for %i °\" %(int(np.round(i*180/np.pi,0))))\n",
+    "    cmd = \"~/ogs_release/bin/ogs \" + filename\n",
+    "    os.system(cmd)\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
All this metadata (like execution count) are not nice for git storage. I'd suggest to remove all of the output, at least.
Would conversion to python be good? Goes like this `jupyter nbconvert --to="python"`

Reply:
Will do